plugins/http/server_intercepter.go

```diff
@@ -59,6 +59,9 @@ func (h *ServerInterceptor) AfterInvoke(invocation operator.Invocation, result .
 	span := invocation.GetContext().(tracing.Span)
 	if wrapped, ok := invocation.Args()[0].(*writerWrapper); ok {
 		span.Tag(tracing.TagStatusCode, fmt.Sprintf("%d", wrapped.statusCode))
+		if wrapped.statusCode >= 400 {
+			span.Error("")
```

Comment:
@mrproliu I feel we should support Error method with no log message, rather than an empty string.
Error message should a log, right?

Reply:
![image](https://github.com/user-attachments/assets/b74b61e2-0eff-47a5-ad6b-4901f64f1d61)
Is that OK?

Reply:
You could ref to Java agent, method name is errorOccured.

Reply:
The error method's parameter is a variable string list, and you can not pass in any methods. Additionally, as I mentioned in [previous comments](https://github.com/apache/skywalking/issues/12882#issuecomment-2556112341), a check for length 0 needs to be added in the Log method.

Reply:
![image](https://github.com/user-attachments/assets/b1f33a96-4517-4721-ac3a-b929f016213e)
![image](https://github.com/user-attachments/assets/8f5287ef-70d5-4c02-9479-43657738b908)
OK?

Reply:
If you have a separate error method, you don't need to support log length=0, aren't you?

Reply:
sorry, I'm lost. Your mean is like this?
![image](https://github.com/user-attachments/assets/85190b75-1597-4b88-bd5f-0e130d9c4274)

Reply:
Yes, and the original error(logs) method is kept unchanged.

Reply:
Done

Reply:
Yes, we can do that, but for unified code management, shouldn't the new method still call the Error method (without passing parameters)? I think both options are fine. @wu-sheng WDYT?

Reply:
It depends. Error(logs) is different from labeling a span in error status.